scripts/circle/build_cocoapods_archive.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+# Create a zip archive of files that will serve as the Cocoapods source.
+
+# Build the universal binary framework.
+make ios
+
+# First add the license file.
+zip ${CIRCLE_ARTIFACTS}/pod.zip LICENSE
```

Comment:
The pod will also need a README (the one that will be displayed on https://cocoapods.org/pods/Tangram-es). We can pick the one that was previously used in the ios-framework repo, since the one we have on the iOS platform folder is not really relevant for direct user of the pod.